lib/api/multiObjectDelete.js

```diff
@@ -205,17 +322,28 @@ function multiObjectDelete(authInfo, request, log, callback) {
 
     return async.waterfall([
         function parseXML(next) {
-            return _parseXml(request.post, (err, quietSetting, objects) => {
-                if (err || objects.length < 1 || objects.length > 1000) {
-                    return next(errors.MalformedXML);
-                }
-                return next(null, quietSetting, objects);
-            });
+            return _parseXml(request.post,
+                (err, quietSetting, objects, itemErrors) => {
```

Comment:
add comment as to what itemErrors are - improperly formed version ID's in requests to delete specific versions of objects?